src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -214,6 +222,7 @@ const InstancePage = (): JSX.Element => {
           <TabContent
             eventKey={0}
             id="instance-page__tabs-processors"
+            ouiaId="processors-content"
```

Comment:
```suggestion
            ouiaId="processors-tab-content"
```

Reply:
I would say we might not need neither content not tab-content as part of ouiaId. The fact that it is a TabContent component should result in a distinctive data-ouia-component-type attribute, so ouiaId needs to distinguish only the instances of this component type (within the surrounding context).
In selectors we'll use both type and id information, so we're not losing anything.